frontend/index.html

```diff
@@ -31,6 +31,12 @@ <h1>Profiles</h1>
                 <option value="36109">36109</option>
                 <option value="36110">36110</option>
             </select>
+            <br>
+            Profile:
+            <select name="profile">
+              <option value="default-docker">default-docker</option>
+              <option value="weak-docker">weak-docker</option>
+            </select>
```

Comment:
Maybe a link to the profiles documentation:
https://github.com/kinvolk/container-escape-bounty/blob/master/Documentation/profiles.md